examples/semantic_search/cohere_wikipedia_ingest.ipynb

```diff
@@ -0,0 +1,346 @@
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "646424\n",
+      "10000\n"
+     ]
+    }
+   ],
+   "source": [
+    "from datasets import load_dataset\n",
+    "lang = \"simple\" # Smaller than the \"en\" dataset\n",
+    "full_dataset = load_dataset(\"Cohere/wikipedia-2023-11-embed-multilingual-v3\", lang)\n",
+    "dataset = full_dataset[\"train\"]\n",
+    "print(len(dataset))\n",
+    "N_DOCS = 10000\n",
+    "dataset = dataset.select(range(N_DOCS)) # Comment this line out to use the full dataset\n",
+    "print(len(dataset))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Wrap these embeddings for LangChain\n",
+    "\n",
+    "LangChain expects a class that will create embeddings on-the-fly, but we have a set of pre-computed embeddings.\n",
+    "This is a wrapper class that bridges the gap."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import langchain_core.embeddings\n",
+    "try:\n",
+    "    from typing import override\n",
+    "except ImportError:\n",
+    "    def override(func):\n",
+    "        return func\n",
+    "\n",
+    "class PrecomputedEmbeddings(langchain_core.embeddings.embeddings.Embeddings):\n",
+    "    @classmethod\n",
+    "    def from_dataset(class_, dataset):\n",
+    "        result = class_()\n",
+    "        result.index = {doc['text']: doc['emb'] for doc in dataset}\n",
+    "        return result\n",
+    "\n",
+    "    @override\n",
+    "    def embed_documents(self, texts):\n",
+    "        # Will throw if text is not in index\n",
+    "        return [self.index[text] for text in texts]\n",
+    "\n",
+    "    @override\n",
+    "    def embed_query(self, query):\n",
+    "        # Will throw if text is not in index\n",
+    "        return self.index[query]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can create our LangChain embeddings object that will work on the Wikipedia corpus.\n",
+    "\n",
+    "If you elected not to use a subset of documents, this will take "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "embeddings = PrecomputedEmbeddings.from_dataset(dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to ApertureDB\n",
+    "\n",
+    "For the next part, we need access to a specific ApertureDB instance.\n",
+    "There are several ways to set this up.\n",
+    "The code provided here will accept ApertureDB connection information as a JSON string.\n",
+    "See our [Configuration](https://docs.aperturedata.io/Setup/client/configuration) help page for more options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! adb config create rag --from-json --active "
```

Comment:
```suggestion
    "! adb config create rag_adb_server_config  --from-json --active "
```

Reply:
I am thinking we should do more descriptive config names for the configs OR how does it play with the one in JSOn already?

Reply:
We will be able to drop the name entirely once https://github.com/aperture-data/aperturedb-python/pull/510 is released.